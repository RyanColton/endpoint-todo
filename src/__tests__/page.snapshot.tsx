import { render } from '@testing-library/react'
import Page from '../app/page'

jest.mock('../app/apiCalls', () => ({
  getTodoList: jest.fn().mockResolvedValue({ json: () => ({
    description: "Run LA marathon",
    dueDate: "2023-03-21T13:30:00.000Z",
    id: "6",
    isComplete: false,
  }) }),
}));
 
it('renders basic layout of todo app with mocked data', async () => {
  const { container } = await render(<Page />)
  expect(container).toMatchSnapshot()
})