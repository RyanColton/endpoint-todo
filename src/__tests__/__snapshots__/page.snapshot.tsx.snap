// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page snapshot renders basic layout of todo app with mocked data 1`] = `<div />`;

exports[`renders basic layout of todo app with mocked data 1`] = `<div />`;
